inc/setup/cachify.redis.nginx.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Setup for Redis.
+ *
+ * @package Cachify
+ */
+
+/* Quit */
+defined( 'ABSPATH' ) || exit;
+?>
+
+<h2><?php esc_html_e( 'Redis setup', 'cachify' ); ?></h2>
+<p><?php esc_html_e( 'Please ensure the following variables are added to your environment', 'cachify' ); ?></p>
```

Comment:
"environment" is not self explaining here. Should be pointed out clearly that the system or process environment is meant here and not something like the _wp-config.php_ or webserver variables.

Reply:
Removed setup instructions for redis in https://github.com/pluginkollektiv/cachify/pull/252/commits/7afb46632f5d3d7de4ff54a456c5546e44717926